.github/workflows/on_pull_request_contrib.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull-Request (Contrib)
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  checks:
+    name: "Checks"
+    if: github.event.pull_request.head.repo.owner.login != 'rerun-io'
+    uses: ./.github/workflows/contrib/checks.yml
+    with:
+      CONCURRENCY: pr-${{ github.event.pull_request.number }}
+      PR_NUMBER: ${{ github.event.pull_request.number }}
```

Comment:
Doesn't really matter, but I feel like it would be more obvious to have these in `on_pull_request` following the non-contrib sections.

Reply:
I wanted the name of the job to be different from the usual `Pull-Request`